distribution/client/src/lib/storage.ts

```diff
@@ -77,11 +77,20 @@ export default class Storage {
     const pluginPath = this.pluginsDirectory();
     const retArray = [];
     plugins.forEach((plugin) => {
+      retArray.push(new Promise((resolve, reject) => {
+        fs.remove(`${pluginPath}/${plugin}`, (err) => {
+          if(err) {
+            logger.debug(`${pluginPath}/${plugin} was not found.`);
```

Comment:
I think we should analyze the error to make the log clearer, for instance:
* if ENOENT then I agree with the change and to **not** fail in case the deletion, in that case debug is probably enough
* in most (all?) other cases, then at least it should be a warning (because if there's a permission issue for whatever, we're likely to crash soon if we really need that directory to be removed)

Reply:
Created https://issues.jenkins-ci.org/browse/JENKINS-54207